kiali/README.md

```diff
@@ -40,22 +40,40 @@ An alternative can be a parallel installation of the upstream chart offering all
 
 1. Run the Helm upgrade command, which installs the chart if not present yet.
     ```bash
```

Comment:
``````suggestion
Run the Helm upgrade command, which installs the chart if not present yet.
```bash
``````

There's only one step here, so the list is not needed + the last sentence in line 46 should be a part of it if anything.

Apply this together with the suggestion in the next comment.